### PR TITLE
Add statement of truth signature property in submit transformer - forms 10-10d and 10-7959c

### PR DESCRIPTION
**Pull request by michaelclement**
## Summary

This PR updates the submit transformers for forms 10-10d and 10-7959c so that they both have a `statementOfTruthSignature` property. 

The property already exists on 10-10d, but needed to be elevated to the top-level of the `data` object sent to the backend. 

10-7959c uses a custom signature block that does not produce that property, so we add it here.

- Adds statementOfTruthSignature to 10-10d submit transformer data sent to backend
- Adds statementOfTruthSignature to 10-7959c submit transformer data sent to backend
- Team: IVC CHAMPVA
- Flipper: NA

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/87181

## Testing done

- Manual

## Screenshots
NA

## What areas of the site does it impact?

IVC forms 10-10d and 10-7959c

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

NA

### CODE REVIEW

File: src/applications/ivc-champva/10-10D/config/submitTransformer.js [hunk 1]
```diff
@@ -188,6 +188,10 @@ export default function transformForSubmit(formConfig, form) {
     dataPostTransform.veteran.address.postalCode || '';
   delete dataPostTransform.veteran.address.postalCode;
 
+  dataPostTransform.certifierRole = transformedData.certifierRole;
```
Comment (michaelclement) [inline src/applications/ivc-champva/10-10D/config/submitTransformer.js:191]: This line will disappear as a change once https://github.com/department-of-veterans-affairs/vets-website/pull/30571 is merged

File: src/applications/ivc-champva/10-10D/config/submitTransformer.js [hunk 2]
```diff
@@ -188,6 +188,10 @@ export default function transformForSubmit(formConfig, form) {
...
+  dataPostTransform.statementOfTruthSignature =
+    transformedData.statementOfTruthSignature;
+
```
Comment (michaelclement) [inline src/applications/ivc-champva/10-10D/config/submitTransformer.js:194]: This adds the property to the top level of our data object sent to backend

File: src/applications/ivc-champva/10-7959C/config/submitTransformer.js [hunk 3]
```diff
@@ -50,6 +50,8 @@ export default function transformForSubmit(formConfig, form) {
   // a status event notification
   copyOfData.primaryContactInfo = getPrimaryContact(copyOfData);
 
+  copyOfData.statementOfTruthSignature = copyOfData.signature;
```
Comment (michaelclement) [inline src/applications/ivc-champva/10-7959C/config/submitTransformer.js:53]: Makes sure we have the signature property that the PDF stamper will be looking for.

**Review by balexandr — DISMISSED** [review]

**Review by cloudmagic80 — DISMISSED** [review]

**Review by rsmithadhoc — DISMISSED** [review]

**Comment by michaelclement** [comment]
Hi, @rsmithadhoc - I had to fix a small merge conflict. Could I have re-approval when you get a minute? Thanks!

**Review by rsmithadhoc — APPROVED** [review]

**Pull request merged**